modules/virt-importing-vm-cli.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-importing-vm-cli_{context}']
+= Importing a Red Hat Virtualization virtual machine with the CLI
+
+You can import a Red Hat Virtualization virtual machine with the CLI by creating the Secret and VirtualMachineImport Custom Resources (CRs).
+
+Optional: You can create a ResourceMapping CR that is separate from the VirtualMachineImport CR. A ResourceMapping CR provides greater flexibility, for example, if you import additional RHV VMs.
+
+.Procedure
+
+. Create a Secret CR to store the RHV Manager credentials:
++
+[source,yaml]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rhv-credentials
+  namespace: default <1>
+type: Opaque
+stringData:
+  ovirt: |
+    apiUrl: "https://<RHVM-FQDN>:8443/ovirt-engine/api" <2>
+    username: admin@internal
+    password: <3>
+    caCert: |
+      -----BEGIN CERTIFICATE-----
+      <4>
+      -----END CERTIFICATE-----
+EOF
+----
+<1> Optional. You can specify a different namespace.
+<2> Specify the FQDN of the RHV Manager.
+<3> Specify the password for `admin@internal`.
+<4> Specify the link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html-single/rest_api_guide/index#obtaining_the_ca_certificate[RHV Manager CA certificate].
+
+. Optional: Create a ResourceMapping CR if you want to separate the resource mapping from the VirtualMachineImport CR:
++
+[source,yaml]
+----
+$ cat <<EOF | kubectl create -f -
+apiVersion: v2v.kubevirt.io/v1alpha1
+kind: ResourceMapping
+metadata:
+  name: resourcemapping-example
+  namespace: default
+spec:
+  ovirt:
+    networkMappings:
+      - source:
+          name: <rhv-logical-network>/<vnic-profile> <1>
+        target:
+          name: pod
+        type: pod
+    storageMappings:
+      - source:
+          name: <rhv-storage-domain> <2>
+        target:
+          name: <target-storage-class> <3>
+EOF
+----
+<1> Specify the RHV logical network and vNIC profile.
+<2> Specify the RHV storage domain.
+<3> Specify the {VirtProductName} storage class. If no storage class is specified, a PVC is created with the default storage class. If a default storage class has not been defined, the PVC is created for a PV with no storage class.
+
+. Create a VirtualMachineImport CR to configure the virtual machine import:
```

Comment:
I don't think a user would do this. And if it is done by accident, the worse that can happen is that the command won't work and they'll have to do it over again. 

Analogous situation: we used callouts long before we had a script that enabled the user to copy a block of code without the callout numbers. We didn't create code blocks with and without callouts in case the user copied the `<1>` by accident. If they included the callout number, the command didn't work. So they did it over again. 

OTOH, I can see some users getting annoyed by not using these combined commands because we are forcing them to do 2 steps instead of one.